.github/workflows/land-blocking.yml

```diff
@@ -5,6 +5,10 @@ on:
   push:
     branches:
       - auto
+    paths-ignore:
```

Comment:
Just realized that we config Homu to wait for signal from this workflow.  @metajack, do you know if Homu would get stuck in waiting or refuse to land the PR in this case?

Reply:
Do you know exactly what happens? Does it just send success immediately, or does it not even show up on the PR at all? I couldn't find anything in the docs besides `the workflow will not run`.

Reply:
It doesn't seem to show up on the PR. Querying the actions/runs API doesn't yield a record of the PR.  I am not sure if there is something happening behind the scene beyond what I can tell.  I will ask in the GH Actions community. 

BTW, what event ("send success immediately") is Homu expecting?

Reply:
homu gets webhook notifications for github checks API changes. So when an action adds a check to the PR or when a check status changes to succeeded or failed, etc.

Homu won't land PRs unless it gets a success status for every check it considers land blocking, so you're right that homu will get stuck if cluster-test is skipped.

An alternative to `paths-ignore` would be to do the paths-ignore filtering yourself in the GHA and then just trivially succeed if all paths match the ignore targets.

Reply:
Yeah, I was originally hoping the `paths-ignore` option would be a quick and dirty hack. DIY filter is probably better as we can use it in CircleCI as well.